crates/hyperqueue/src/client/output/cli.rs

```diff
@@ -536,49 +541,52 @@ impl Output for CliOutput {
         );
     }
 
-    fn print_job_output(&self, job: JobDetail, task_selector: &Selector, output_stream: &OutputStream) -> anyhow::Result<()> {
+    fn print_job_output(
+        &self,
+        job: JobDetail,
+        task_selector: &Selector,
+        output_stream: &OutputStream,
```

Comment:
Let's just pass `OutputStream` and `Selector` by value, no need for references here.